ckan/cli/db.py

```diff
@@ -129,6 +129,7 @@ def duplicate_emails():
                     fg=u"green", bold=True)
     except Exception as e:
         tk.error_shout(e)
+    click.secho(u"No duplicate emails found", fg=u"green")
```

Comment:
Hm, this line is going to be executed every time, even if there **are** duplicates, right?

Reply:
:facepalm: